src/pages/index.tsx

```diff
@@ -32,39 +34,74 @@ const Home: NextPage = () => {
   const { angle, ref } = useElementMouseRelativeAngle();
 
   return (
-    <div className="flex h-screen w-screen flex-col">
-      <Navbar />
-      <div className="flex w-screen grow flex-col items-center justify-center gap-16 p-12 md:flex-row xl:gap-32">
-        <div className="flex w-full flex-col items-center justify-center gap-4 md:w-[300px] md:items-start">
-          <div className="text-5xl font-bold">🪂</div>
-          <div className="text-5xl font-bold">when2meet</div>
-          <div className="text-orange-elevated mt-[-4px] text-6xl font-bold">
-            Elevated
-          </div>
-          <div className="mt-2 text-gray-500">
-            Open-source, easy-to-use, free forever
-          </div>
-        </div>
-        <div
-          ref={ref}
-          className="bg-colorful drop-shadow-pink h-[250px] w-full rounded-[8px] p-[2px] sm:max-w-[350px]"
-          style={
+    <>
+      <Head>
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
But where is the favicon file)?

Reply:
can you make the favicon to be our logo (the one used for this github organization)?